.yarnrc.yml

```diff
@@ -11,9 +13,3 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
     spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
-  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
```

Comment:
All plugins are enabled by default, so there is no need to store these in the repo: https://github.com/yarnpkg/berry/issues/3591